daemon/logger/logger.go

```diff
@@ -15,4 +21,6 @@ type Logger interface {
 	Log(*Message) error
 	Name() string
 	Close() error
+	Read() (io.Reader, error)
```

Comment:
@LK4D4 alright, fixing this as `GetReader`. Great to hear there will be multiple drivers with reading capabilities, it makes more sense to push these logic here down to the driver interface.